### DISCUSSION TITLE
/api/_content/query very slow

### DISCUSSION THREAD

**Discussion by jianchengwang** [nuxt/content#1699] [Q&A]
hello guys, I hear `content v2` is supported, so I just upgrade my blog site from `content v1` to `content v2`. But I found that even with just 2 markdown files, query content spend 3 seconds, and at first, If tens of markdown files, it will take more long time, maybe 2 min or more. If this can't be solved, I don't know how to develop.  But `content v1` everything works well and query very fast. So every body found this question or just my config problems

```
http://localhost:3000/api/_content/query?_params={%22only%22:[%22tags%22],%22where%22:[{%22_path%22:%22--REGEX+/^%5C%5C/tech/%22}],%22sort%22:[{%22_file%22:1,%22$numeric%22:true}]}
```

![image](https://user-images.githubusercontent.com/15951975/202962610-283bacb7-6693-423e-a5a3-2002ca77aeec.png)

**Comment by Barbapapazes** [comment]
Yes, currently in process in #1651
